src/js/components/FloatingElement.jsx

```diff
@@ -422,12 +422,16 @@ export class FloatingElement extends Component {
     }
 
     componentRendered() {
-        // As soon as possible, we need to re-calculate our position
-        this.validatePositioningScheduled = true;
-        window.requestAnimationFrame(() => {
-            this.validatePositioningScheduled = false;
+        if (this.props.positionImmediate) {
```

Comment:
Not opposed to that at all. We need to double-check the opacity hack; I seem to recall Safari doing something really weird like giving keyboard focus to the elements in the menu even though opacity is 0; that's why I had to switch to creating/destroying the DOM when the menu opened and closed.